docs/sdk-and-tools/erdjs/migration-guides.md

```diff
@@ -9,9 +9,156 @@ This tutorial will guide you through the process of migrating from one major ver
 Make sure you have a look over the [cookbook](/sdk-and-tools/erdjs/erdjs-cookbook), in advance.
 :::
 
+## Migrate **erdjs** from v11.x to v12 (October of 2022)
+
+**erdjs v12** brings a series of breaking changes that we will explain below.
+
+### Transaction version and options
+
+Starting with **erdjs v12**, support for `GuardAccount` feature will be available to use. This means that another option for transaction has been activated. Therefore, now we have: 
+
+```
+export const TRANSACTION_OPTIONS_DEFAULT = 0;
+export const TRANSACTION_OPTIONS_TX_HASH_SIGN = 1;
+export const TRANSACTION_OPTIONS_TX_GUARDED = 2;
+```
+Given this addition it means that previously `export const TRANSACTION_VERSION_TX_HASH_SIGN = 2` becomes obsolete because a *version 2* transaction will support more than one option. That's why we changed this constant to:
+
+```
+export const TRANSACTION_VERSION_WITH_TX_OPTIONS = 2;
+```
+As a consequence, in `src/networkParams.ts` the method previously named `withTxHashSignVersion()` will become:
+
+```
+    /**
+     * Creates a TransactionVersion object with the VERSION setting for enabling options
+     */
+    static withTxOptions(): TransactionVersion {
+        return new TransactionVersion(TRANSACTION_VERSION_WITH_TX_OPTIONS);
+    }
+```
+
+Also changes on [elrond-sdk-erdjs-walletcore](https://github.com/ElrondNetwork/elrond-sdk-erdjs-walletcore) have been implemented and will be made available with **v3.0.1** release (see [details](https://docs.elrond.com/sdk-and-tools/erdjs/erdjs-migration-guides#elrond-sdk-erdjs-walletcore---v301-release) below).
+
+### Guarded Transaction
+
+After setting a guardian (`SetGuardian`) and sending `GuardAccount` builtin transaction to Elrond network, only guardian co-signed transactions will be accepted by the *guarded* account. For supporting this, two new fields have been added to the transaction proto (Guardian Address and Guardian Signature):
```

Comment:
Perhaps it would be better to provide less details on this page?

E.g. `after setting a guardian for an account` should be more than enough. Then, possibly add a link to the actual documentation of the `guardians` feature.

Reply:
Changing the `proto` definition is a concept that erdjs users should normally not care about. I do not know whether we should document this or not.